listenbrainz/webserver/views/test/test_user.py

```diff
@@ -39,6 +38,14 @@ def setUp(self):
             'INFLUX_DB_NAME': current_app.config['INFLUX_DB_NAME'],
         })
 
+        db_user.get_or_create(1, 'iliekcomputers')
+        user = db_user.get_by_mb_id('iliekcomputers')
```

Comment:
We don't need to have this line twice, `get_or_create` will also return the db row, so we can just use `user = get_or_create(blah)`

Reply:
Okay, fixed.